mix/stdc/config.sol

```diff
@@ -0,0 +1,45 @@
+//sol Config
+// Simple global configuration registrar.
+// @authors:
+//   Gav Wood <g@ethdev.com>
+#require mortal
```

Comment:
This will not compile anymore. Please use `import "qrc:///stdc/mortal.sol";` - or whatever you use as source unit identifier when you call the compiler.